pkg/api/types.go

```diff
@@ -2793,6 +2793,11 @@ type ExportOptions struct {
 	Export bool `json:"export"`
 	// Should the export be exact.  Exact export maintains cluster-specific fields like 'Namespace'
 	Exact bool `json:"exact"`
+	// If specified:
+	// - if unset, then the result result is returned from remote storage based on quorum-read flag;
+	// - if it's 0, then we simply return what we currently have in cache, no guarantee;
+	// - if set to non zero, then the result is as fresh as given rv.
+	ResourceVersion string
```

Comment:
Fixing this is part of my server API group issue.  There are disadvantages to mixing export and get because I don't believe that all exports are created equal yet.  However, it's likely we have a core Get schema + a set of optional flags that vary.   I'd prefer to not to push too far on that with this Pr